lib/charms/opensearch/v0/opensearch_base_charm.py

```diff
@@ -367,95 +448,157 @@ def _is_tls_fully_configured(self) -> bool:
 
         return self._are_all_tls_resources_stored()
 
-    def _start_opensearch(self, event: EventBase) -> None:
+    def _start_opensearch(self, event: EventBase) -> None:  # noqa
```

Comment:
please specify which exactly check should be disabled
```suggestion
    def _start_opensearch(self, event: EventBase) -> None:  # noqa: E501
```